src/ansys/fluent/core/__init__.py

```diff
@@ -70,6 +70,7 @@ def version_info() -> str:
     appname="ansys_fluent_core", appauthor="Ansys"
 )
 EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
+CONTAINER_MOUNT_PATH = os.getcwd()
```

Comment:
One concern that I have with this approach, is that the user may import pyfluent, change working directories, and then expect to work with Fluent in the new working directory, which I would argue is what PyFluent should be supporting by default.  

I believe that would be the main argument against this approach setting `CONTAINER_MOUNT_PATH` on import, compared to just evaluating dynamically it at runtime.

Reply:
Good point. We can get that working with the module variable also: set `CONTAINER_MOUNT_PATH = None` here and later `host_mount_path = pyfluent.CONTAINER_MOUNT_PATH or os.getcwd()`.

Just trying to avoid env var usage by users, unless it is the only way to do something.

Reply:
Done. Thank you.